src/app/pages/administration/jobs/user-triggers-view.jsx

```diff
@@ -30,7 +30,7 @@ const useStyles = makeStyles({
     maxHeight: '64vh',
   },
   cardSpacing: {
-    marginTop: '32px',
+    marginTop: '0',
```

Comment:
Is this an override? If the default is a margin-top of 0 do we need this class at all?